### ISSUE TITLE
Downloading raw stack trace files for native crashes are missing the binary image offsets + load addresses as they appear in the web ui

### ISSUE THREAD

**Issue by idosun** [getsentry/sentry-native#876]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. go to any native issue 
2. display as “Raw stack trace” + “Unsymbolicated”
3. click download
4. observe that image load addresses are missing from the bottom of the file, but present in the web UI

![image](https://github.com/getsentry/sentry/assets/6572704/9a52ad04-b8cd-46a4-ae45-d9111cec492a)

### Expected Result

Downloaded stack trace should contain the Binary images load + offset addresses 

### Actual Result

EOF right after the list of thread dumps

### Product Area

Issues

**Comment by supervacuus** [comment]
Hi @idosun, this isn't a client-side issue but rather an issue (or deliberate decision, I don't know) of the sentry backend. 

I can see that it was moved here. @Fwang36, can you tell me how the Native SDK can help with the issue? All client-side data ended up in the backend, but the particular download link produced a response with the binary images excluded.

cc: @kahest, @Swatinem

**Comment by kahest** [comment]
> Hi @idosun, this isn't a client-side issue but rather an issue (or deliberate decision, I don't know) of the sentry backend.
> 
> I can see that it was moved here. @Fwang36, can you tell me how the Native SDK can help with the issue? All client-side data ended up in the backend, but the particular download link produced a response with the binary images excluded.
> 
> cc: @kahest, @Swatinem

I agree, this is not something we can solve on the SDK side. @rachrwang is this something the Issues platform team could look into?

**Comment by idosun** [comment]
Correction, this is in the context of Apple crashes captured by the Cocoa SDK. As an example: https://sentry-sdks.sentry.io/issues/4308453386/?project=5428557&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=30d&stream_index=17

**Comment by idosun** [comment]
@rachrwang This definitely seems related to issue platform - more detail in the [internal Jira ticket](https://getsentry.atlassian.net/browse/ISSUE-1680)

**Comment by supervacuus** [comment]
Yes, thank you, @idosun, for the updates. Independent of where the crashes are coming from, it's not the SDKs causing this problem. But the hint towards the Cocoa SDK could answer why I don't see binary images in the "Native" issue, but you do.

**Comment by Fwang36** [comment]
I'm transferring back to the main repo.  Sorry for the mixup!
